.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
```

Comment:
If this section is meant to be deleted, make it a comment?

Reply:
Done.